builtin/settingtypes.txt

```diff
@@ -819,6 +819,9 @@ server_url (Server URL) string https://minetest.net
 #    Automatically report to the serverlist.
 server_announce (Announce server) bool false
 
+#    Make local servers visible to local clients.
+serverlist_lan (Show local servers) bool true
```

Comment:
i'd called it serverlist_lan_discovery

Reply:
I am all for renaming it to whatever makes the most sense, does anyone else have any other suggestions?

Reply:
Another option would be to not create a new setting but to just change server_announce from a bool to a list of options like:
    None -- No announcing
    Lan Only -- Only visible to local clients
    Global -- Announces to the regular server list

Reply:
I really like the idea of using server_announce instead. I think I will do it that way if no one would object to that.

Reply:
server_announce is more generic and would mean we announce to global server list. I'd call it server_lan_announce then